loader/base_loader.py

```diff
@@ -377,6 +389,7 @@ def _upload_tagged_cloud_file_to_dss_by_copy(self, source_bucket: str,
 
             timeout = time.time() + timeout_seconds
             wait = 1.0
+            # FIXME busy wait could hopefully be replaced with asyncio
```

Comment:
That's exactly *why* it can be used. any time there is a local wait in python another async func could be running. I'm not sure how often this code path is even used though. That research needs to be done to decide if adding asyncio here would actually make any substantial difference.